src/dotnet/App.Maui/Platforms/Android/AndroidWebChromeClient.cs

```diff
@@ -67,7 +67,8 @@ public AndroidWebChromeClient(WebChromeClient client, ComponentActivity activity
         _client = client;
         _fileChooser = fileChooser;
 
-        _requestPermissionLauncher = _activity.RegisterForActivityResult(
+        // ReSharper disable once NullCoalescingConditionIsAlwaysNotNullAccordingToAPIContract
+        _requestPermissionLauncher ??= _activity.RegisterForActivityResult(
```

Comment:
Why do you need `??=` operator here?

Reply:
When WebChromeClient is getting recreated we should keep single registration within the same activity - otherwise it will fail, e.g. we should register before the activity reach STARTED state